src/core/opamCmd.mli

```diff
@@ -0,0 +1,13 @@
+(**************************************************************************)
+(*                                                                        *)
+(*    Copyright 2025 Kate Deplaix                                         *)
+(*                                                                        *)
+(*  All rights reserved. This file is distributed under the terms of the  *)
+(*  GNU Lesser General Public License version 2.1, with the special       *)
+(*  exception on linking described in the file LICENSE.                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+val of_string : string -> string list
```

Comment:
Is there a reason to add a new module instead of adding it in `OpamStd`?

Reply:
In my opinion `OpamStd` should only be an overlay over the standard library, a `Cmd` module doesn't fit that. Also the definition is big enough to warrant its own module, and in the future i'd also like to add more functions to this module like `to_string`.